sqlmesh/core/config/model.py

```diff
@@ -31,7 +28,6 @@ class ModelDefaultsConfig(BaseConfig):
     cron: t.Optional[str] = None
     owner: t.Optional[str] = None
     start: t.Optional[TimeLike] = None
-    batch_size: t.Optional[int] = None
```

Comment:
Removing this value, since it breaks the project if set anyway:
```
Error: 1 validation error for SqlModel
batch_size
  Extra inputs are not permitted [type=extra_forbidden, input_value=1, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden at '/Users/izeigerman/github/sqlmesh/examples/sushi/models/marketing.sql'
```
It assumes that the `batch_size` is the top-level model attribute.